resources/quality/builder_premium/bp_PLA_Coarse_Quality.inst.cfg

```diff
@@ -7,9 +7,8 @@ definition = builder_premium_small
 type = quality
 quality_type = coarse
 material = generic_pla_175
-setting_version = 4
+setting_version = 3
```

Comment:
Can you change all setting_version fields to 4? They were updated for the 3.1 release.